src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests.cs

```diff
@@ -28,7 +28,7 @@ public AddUsingTests(ITestOutputHelper logger)
         {
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(IsRelease), Reason = ConditionalSkipReason.TestIsTriggeringMessagePackIssue)]
```

Comment:
I'm not sure if all tests are impacted, but this was a blanket search/replace for this entire file.

Reply:
I think thats reasonable. I am still fuzzy of the value of these tests on a 32 bit run. Perhaps vs4mac(?)